dependencies/python-hwinfo/Makefile

```diff
@@ -0,0 +1,23 @@
+NAME = python-hwinfo
+VERSION = 0.1.6
+
+all: srpm
+
+clean:
+	rm -rf dist/
+	rm -rf $(NAME)-$(VERSION).tar.gz
+	rm -rf $(NAME)-$(VERSION)-1.src.rpm
+	rm -fr *.log
+
+dist:
+	git clone https://github.com/rdobson/python-hwinfo $(NAME)-$(VERSION)
+	tar -zcvf $(NAME)-$(VERSION).tar.gz $(NAME)-$(VERSION)
+	rm -fr $(NAME)-$(VERSION)
+
+srpm: dist
+	fedpkg --dist epel7 srpm
+
+rpm:  srpm
+	mock -r epel-7-x86_64 rebuild $(NAME)-$(VERSION)-1.src.rpm --resultdir=.
```

Comment:
Similar that above comment. This pull request has in name "el6 support". I think that there should be ```epel-6-x86_64``` instead of ```epel-7-x86_64```.